pkgs/development/tools/language-servers/pylyzer/default.nix

```diff
@@ -0,0 +1,56 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+, git
+, python3
+, makeWrapper
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "pylyzer";
+  version = "0.0.26";
+
+  src = fetchFromGitHub {
+    owner = "mtshiba";
+    repo = "pylyzer";
+    rev = "v${version}";
+    hash = "sha256-ZEmTSSYHQWk0IVJXlrtGb+j2hbb9ZtDLCtajOR7BMoU=";
+  };
+
+  cargoHash = "sha256-/QMzPvLcAjpai2YX58+YM/+KhYZRuK59hPYAEHeTTa4=";
+
+  nativeBuildInputs = [
+    git
+    python3
+    makeWrapper
+  ];
+
+  buildInputs = [ python3 ];
```

Comment:
```suggestion
  buildInputs = [
    python3
  ] ++ lib.optionals stdenv.isDarwin [
    darwin.apple_sdk.frameworks.Security
  ];
```

Reply:
Thanks, I missed the build failure on darwin.